File: LeetCode/Easy/0232-implement-queue-using-stacks/0232-implement-queue-using-stacks.java
class MyQueue {
    private ArrayList<Integer> q;
    public MyQueue() {
        q = new ArrayList<>();
    }
    
    public void push(int x) {
        q.add(x);
    }
    
    public int pop() {
        int ret = q.get(0);
        q.remove(0);
        return ret;
    }
    
    public int peek() {
        return q.get(0);
    }
    
    public boolean empty() {
        return q.size() == 0;
    }
}

/**
 * Your MyQueue object will be instantiated and called as such:
 * MyQueue obj = new MyQueue();
 * obj.push(x);
 * int param_2 = obj.pop();
 * int param_3 = obj.peek();
 * boolean param_4 = obj.empty();
 */

File: LeetCode/Easy/0232-implement-queue-using-stacks/0232-implement-queue-using-stacks.py
class MyQueue:

    def __init__(self):
        self.q = []

    def push(self, x: int) -> None:
        self.q.append(x)

    def pop(self) -> int:
        ret = self.q[0]
        self.q = self.q[1:]
        return ret

    def peek(self) -> int:
        return self.q[0]

    def empty(self) -> bool:
        return not self.q


# Your MyQueue object will be instantiated and called as such:
# obj = MyQueue()
# obj.push(x)
# param_2 = obj.pop()
# param_3 = obj.peek()
# param_4 = obj.empty()